test/smoke/src/areas/positron/console/consoleInput.test.ts

```diff
@@ -81,10 +81,67 @@ cat(sprintf('Hello %s!\n', val))`;
 
 					await app.workbench.positronConsole.sendEnterKey();
 
-					await app.workbench.positronConsole.waitForConsoleContents((contents) => contents.some((line) => line.includes('Hello John Doe!')) );
-				}).toPass({timeout: 60000});
+					await app.workbench.positronConsole.waitForConsoleContents((contents) => contents.some((line) => line.includes('Hello John Doe!')));
+				}).toPass({ timeout: 60000 });
 
 			});
+
+			it('R - Can use `menu` to select alternatives', async function () {
```

Comment:
@jonvanausdeln I think we also need a test rail id for this one and the one below